### PR TITLE
fixes bug 1164465 - psycopg2 2.6

### PR DESCRIPTION
**Pull request by peterbe**
Going to r someone as soon as I see this working once in travis.

### CODE REVIEW

**Comment by rhelmer** [comment]
This should work, *but* wouldn't fix the deadlock problem we're seeing in AWS load-testing since psycopg2 2.6.1 isn't out yet :(

@peterbe - should we use a git sha until 2.6.1 is released?

**Comment by peterbe** [comment]
Oh. I see! That's problematic. I hate when this happens. But we have little choice don't we. This bug you mention that 2.6.1 fixes, is hurting us actively this week already.

**Comment by peterbe** [comment]
So, according to this, it is a release http://initd.org/psycopg/docs/news.html

**Comment by peterbe** [comment]
The two tickets under 2.6.1 on http://initd.org/psycopg/docs/news.html don't mention https://github.com/psycopg/psycopg2/pull/290

**Comment by rhelmer** [comment]
@peterbe yeah... 2.6.1 isn't on pypi it looks like, but it doesn't contain the fix it won't help us anyway.

**Comment by rhelmer** [comment]
OK so it looks like 2.6.1 *does* contain this fix, but it's not posted on pypi yet.

File: requirements.txt [hunk 1]
```diff
@@ -47,8 +47,15 @@ lxml==3.4.1
 pgxnclient==1.2.1
 # sha256: Zm_VIvShBZ1bi0mW5Xb6eGL1BVKOEc8e9VRjEcYMPk0
 pika==0.9.8
-# sha256: xsMwyYqwjFY58MzX4hG4Kzo58AGBPc6mu4JKmdKzT-I
-psycopg2==2.5.5
+
+# psycopg2 2.6.1 isn't out on pypi at the time of writing and we can't
+# wait so we'll point to the sha for now.
+# Once 2.6.1 on pypi (see
+# https://github.com/psycopg/psycopg2/issues/324)
```
Comment (rhelmer) [inline requirements.txt]: Good idea to comment this

**Comment by rhelmer** [comment]
r+ from me, if it passes travis - I am testing on AWS as we speak as well.

**Comment by peterbe** [comment]
I'm going to rebase this once it finishes (if it finishes) if you don't mind waiting a bit. I don't want that bug to close until we can get off the github sha.

**Comment by dvarrazzo** [comment]
FYI, 2.6.1 is now released on PyPI, thank you for your patience.

**Pull request merged**